.mk/vpp_agent.mk

```diff
@@ -1,9 +1,9 @@
 ARCH ?= $(shell uname -m)
 ifeq (${ARCH}, x86_64)
-  export VPP_AGENT=ligato/vpp-agent:v1.8.1
-  export VPP_AGENT_DEV=ligato/dev-vpp-agent:v1.8.1
+  export VPP_AGENT=ligato/vpp-agent:latest
```

Comment:
Do we know if there is going to be `v2.0.0` or similar tag? Having `latest` here sounds like a potential source of instabilities.

Reply:
Nope, they not have docker images for 2.0 yet.